docs/protocol.md

```diff
@@ -61,7 +63,11 @@ enums used.
   'size'                   =>  8,
   'message_counter'        =>  9,
   'correlation_id'         => 10,
-  'complete_message_count' => 11
+  'complete_message_count' => 11,
```

Comment:
We have many enums repeated in multiple events. For example file, line number, payload, timestamp, event_type appear in other event types also.